src/os/ogc/query/ogcquery.js

```diff
@@ -0,0 +1,141 @@
+goog.module('os.ogc.query.OGCQuery');
+
+goog.require('os.ui.ogc.OGCListUI'); // touch the <ogclist> directive to get it to inject...
+goog.require('plugin.areadata'); // touch the AreaData plugin constants to bring the code into core OS
```

Comment:
Plugin references should never be added outside of plugins and `MainCtrl`.

Reply:
Removed.